src/crosschain/types.ts

```diff
@@ -91,7 +98,7 @@ export type Config = {
     revertableAddress: Partial<Record<ChainId, string>> & { default: string }
     limits: SwapLimit[]
     chains: ChainConfig[]
-    refundAddress: string
+    refundAddress: Address | BtcAddress
```

Comment:
now we use `refundAddress` for BTC refunds only

Reply:
```
src/crosschain/config/mainnet.ts(120,5): error TS2322: Type '"0xd99ac0681b904991169a4f398B9043781ADbe0C3"' is not assignable to type 'BtcAddress'.

src/crosschain/config/dev.ts(22,5): error TS2322: Type '"0x1b5D6DDF6086Bb06616f58274F894099c31e9DB5"' is not assignable to type 'BtcAddress'.
```

Reply:
Or, should this field be moved to `BtcConfig` ?

Reply:
I mixed up two places. in the config it should be EvmAddress, but in SwapExactInParams should be BtcAddress only.
Will rename in the future to avoid confusion.